packages/dev/docs/src/Layout.js

```diff
@@ -148,6 +148,24 @@ function Page({children, currentPage, publicUrl, styles, scripts}) {
         <meta property="og:image" content={`https://${TLD}${heroImage}`} />
         <meta property="og:description" content={description} />
         <meta property="og:locale" content="en_US" />
+        <script
+          type="application/ld+json"
+          dangerouslySetInnerHTML={{__html: `
+            {
+              "@context": "http://schema.org",
+              "@type": "Article",
+              "author": "Adobe Inc",
+              "headline": "${currentPage.title}",
+              "description": "${description}",
+              "image": "${`https://${TLD}${heroImage}`}",
+              "publisher": {
+                "@type": "Organization",
+                "url": "https://www.adobe.com",
+                "name": "Adobe",
+                "logo": "https://www.adobe.com/favicon.ico"
+              }
+            }
+          `}} />
```

Comment:
lol so much duplication. google doesn't pick up the meta tags already? 😞

Reply:
lol...welcome to seo-land.